.changeset/bright-snakes-punch.md

```diff
@@ -0,0 +1,17 @@
+---
+"@rocket.chat/meteor": patch
+"@rocket.chat/uikit-playground": patch
+"@rocket.chat/fuselage-ui-kit": patch
+"@rocket.chat/gazzodown": patch
+"@rocket.chat/livechat": patch
+"@rocket.chat/ui-avatar": patch
+"@rocket.chat/ui-client": patch
+"@rocket.chat/ui-composer": patch
+"@rocket.chat/ui-contexts": patch
+"@rocket.chat/ui-video-conf": patch
+"@rocket.chat/ui-voip": patch
+"@rocket.chat/web-ui-registration": patch
+"@rocket.chat/ui-theming": patch
+---
+
+Bumps fuselage and related packages versions to use the most recent releases of each package, especially the fix for the missing track of the fuselage slider component
```

Comment:
Since this is a chore, I don't think we need the changeset 😉

Reply:
Although this is marked as a chore this PR will fix a bug on the user preferences page together with the package bump and since the actual fix PR for this is on fuselage I've added this changeset so that our end-users are aware of the fix.